Dockerfiles/midstream/run_tests.py

```diff
@@ -6,43 +6,62 @@
 import subprocess
 from os import path
 
+OS_CWD = os.getcwd()
+# global variables set 
+OPERATOR_DIR=OS_CWD+"/operator_dir/"
+OPERATOR_WORK_DIR=OS_CWD+"/test_operator_work_dir/"
+WORK_DIR=OS_CWD+"/output/"
+PLAYBOOKS_DIR="/operator-test-playbooks/"
```

Comment:
-1 for storing playbooks in top level directory
what was wrong with `/project` ?

Reply:
What would be the problem of creating it inside the top directory. ?? 
I am just curious. 
I created the operator-test-playbooks in the top directory inorder to shorten the path.

Reply:
It is not a huge problem it is just very unusual to store data in top level dir.
`/project/operator-test-playbooks` is bit better but people usually store random staff in `/opt`
which is quite short and it adds just 4 charaters